Docker/Jenkins-Worker/Dockerfile

```diff
@@ -8,7 +8,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN set -xe && apt-get update && apt-get install -y apt-utils dnsutils python python-setuptools python-dev python-pip python3 python3-pip build-essential zip unzip less vim gettext-base
 RUN set -xe && python -m pip install awscli --upgrade && python -m pip install pytest --upgrade && python -m pip install PyYAML --upgrade && python -m pip install lxml --upgrade
 RUN set -xe && python3 -m pip install pytest --upgrade && python3 -m pip install PyYAML --upgrade
-RUN set -xe && python -m pip install yq --upgrade && python3 -m pip install yq --upgrade
+RUN set -xe && python -m pip install yq --upgrade && python3 -m pip install yq --upgrade && python3 -m pip install pandas --upgrade
```

Comment:
I think we may have been trying to avoid pandas recently https://github.com/uc-cdis/gen3sdk-python/pull/83#pullrequestreview-638090763

Reply:
Maybe doesn't matter in this context, but not sure if there's a way to do the work without it to avoid the relatively large dependency/need.

Reply:
This is used for release automation inside our Jenkins.
So there are no concerns in terms of service / customer-facing releases... this is for internal productivity tooling only.
We don't have any concerns about performance or .. the size of this image.

Reply:
👍